apps/settings/elements/improve_browser_os.html

```diff
@@ -41,34 +38,31 @@ <h2 data-l10n-id="performanceAndUsageData">Performance And Usage Data</h2>
       </ul>
 
       <header>
-        <h2 data-l10n-id="crashReports">Crash Reports</h2>
+        <h2 data-l10n-id="crashReports"></h2>
       </header>
       <ul>
         <li class="description">
           <p>
-            <span data-l10n-id="crashReportInfo">
-              Sending Mozilla a report when a crash occurs helps us fix the problem
-              for everyone. Reports are sent over Wi-Fi only.
-            </span>
-            <a href="#crashReports" data-l10n-id="learn-more" class="link-text">Learn More</a>
+            <span data-l10n-id="crashReportInfo"></span>
+            <a href="#crashReports" data-l10n-id="learn-more" class="link-text"></a>
```

Comment:
There's no href here. I would think either this and the above link would both have hrefs or neither would.